internal/schema.go

```diff
@@ -141,6 +141,24 @@ type IDPEntityDescriptor struct {
 	ASSERTION        string           `xml:"xmlns:assertion,attr"`
 	EntityId         string           `xml:"entityID,attr"`
 	IDPSSODescriptor IDPSSODescriptor `xml:"IDPSSODescriptor"`
+	Organization Organization `xml:"Organization"`
```

Comment:
@sccalabr 
Add Organization and ContactPerson as a pointer in the Struct with `omitempty` flag as these are optional
If values for these fields are not set then in generated XML these fields XML will not come